### PR TITLE
change button style

### PR DESCRIPTION
**Pull request by kal2944**
- changed contact button style
- fixed the header
- Made the hamburger interactive
- the hamburger icon converted into (x) when pressed and menu items appear 
- the hamburger closes when pressed again or any of the menu items are clicked

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -4,33 +4,85 @@
...
+}
+
+.navbar {
   width: 100%;
-  height: 4rem;
-  padding: 1rem 1.5rem;
+  padding: 3% 2%;
+  font-size: 20px;
+  position: fixed;
 }
```
Comment (mukhammadrizooff) [inline style.css]: - [ ] [REQUIRED] Good job so far! Your mobile menu button and other elements are working very well, but you have not designed the mobile menu like a [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122). Please recheck the design in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) and give the required style.

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @kal2944, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Good implementations ✔️
- Toggle menu working well ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -12,21 +12,32 @@
...
+
+        <ul class="menu">
+          <li class="nav-item link1">
+            <a href="#projects">Portfolio</a>
+          </li>
+          <li class="nav-item link2">
+            <a href="#about-me">About</a>
+          </li>
+          <li class="nav-item link3">
+            <a href="#contact-form">Contact</a>
```
Comment (tufoinnkuo10) [inline index.html:28]: - [ optional] kindly add the font weight and size of the links so they follow the figma design. great job

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @kal2944 

Your project is complete! Great job for making the required changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- mobile menu works as expected
- figma design followed
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (kal2944) [reply index.html:28]: okay, I will arrange it, thank you!

**Pull request merged**